protocols/raft/src/main/java/io/atomix/protocols/raft/proxy/impl/RaftProxyConnection.java

```diff
@@ -275,9 +275,15 @@ protected MemberId next() {
     }
 
     if (!selector.hasNext()) {
```

Comment:
For connections with a `LEADER` strategy, a single failed request can cause a message to fail. This forces all nodes to be contacted after the leader before failing a request.